### PR TITLE
vote9: Replacing the coin toss by a random decision of the referee

### PR DESCRIPTION
**Pull request by babaev1**
In the physical competition, a coin is tossed and the winner gets to decide whether they want to pick a side to play at for the first half time, or whether they want to have the first kick-off. In the virtual competition, both decisions are taken randomly by the referee. The proposal would be to replace the coin toss with the random decision of the referee for the physical competition as well (both for the regular tournament and penalty shoot-outs).

### CODE REVIEW

File: Section_I/law8.tex [hunk 1]
```diff
@@ -47,9 +47,9 @@
 Before a kick-off at the start of the match or extra time 
 
 \begin{itemize}
-\item a coin is tossed and the team that wins the toss decides which goal it will attack in the first half of the match.
-\item the other team takes the kick-off to start the match. 
-\item the team that wins the toss takes the kick-off to start the second half of the match.
+      \item \removed{a coin is tossed and the team that wins the toss decides which goal it will attack} \added{referee randomly decides which team will attack which goal} in the first half of the match.
```
Comment (MaikePaetzel) [inline Section_I/law8.tex:50]: Add "the" in front of "referee"

File: Section_I/law8.tex [hunk 2]
```diff
@@ -47,9 +47,9 @@
...
+      \item \removed{the other} \added{referee randomly decides which} team takes the kick-off to start the match.
```
Comment (MaikePaetzel) [inline Section_I/law8.tex:51]: add "the" in front of "referee"

File: Section_I/law8.tex [hunk 3]
```diff
@@ -47,9 +47,9 @@
 Before a kick-off at the start of the match or extra time 
 
 \begin{itemize}
-\item a coin is tossed and the team that wins the toss decides which goal it will attack in the first half of the match.
-\item the other team takes the kick-off to start the match. 
-\item the team that wins the toss takes the kick-off to start the second half of the match.
+      \item \removed{a coin is tossed and the team that wins the toss decides which goal it will attack} \added{referee randomly decides which team will attack which goal} in the first half of the match.
+      \item \removed{the other} \added{referee randomly decides which} team takes the kick-off to start the match. 
+      \item the team that \removed{wins the toss} \added{is other from team taking kick-off at first half} takes the kick-off to start the second half of the match.
```
Comment (MaikePaetzel) [inline Section_I/law8.tex:52]: Use the wording from the virtual rule book here: "the team that was not given kick-off in the first half of the match takes the kick-off to start the second half of the match."

**Review by MaikePaetzel — CHANGES_REQUESTED** [review]

**Comment by babaev1** [comment]
Resolved

  
>Среда, 2 февраля 2022, 16:18 +03:00 от MaikePaetzel ***@***.***>:
> 
> 
***@***.*** requested changes on this pull request.
>----------------------------------------------------------------------
>In  Section_I/law8.tex :
>> @@ -47,9 +47,9 @@
> Before a kick-off at the start of the match or extra time 
> 
> \begin{itemize}
>-\item a coin is tossed and the team that wins the toss decides which goal it will attack in the first half of the match.
>-\item the other team takes the kick-off to start the match. 
>-\item the team that wins the toss takes the kick-off to start the second half of the match.
>+      \item \removed{a coin is tossed and the team that wins the toss decides which goal it will attack} \added{referee randomly decides which team will attack which goal} in the first half of the match.
>
>Add "the" in front of "referee"
>----------------------------------------------------------------------
>In  Section_I/law8.tex :
>> @@ -47,9 +47,9 @@
> Before a kick-off at the start of the match or extra time 
> 
> \begin{itemize}
>-\item a coin is tossed and the team that wins the toss decides which goal it will attack in the first half of the match.
>-\item the other team takes the kick-off to start the match. 
>-\item the team that wins the toss takes the kick-off to start the second half of the match.
>+      \item \removed{a coin is tossed and the team that wins the toss decides which goal it will attack} \added{referee randomly decides which team will attack which goal} in the first half of the match.
>+      \item \removed{the other} \added{referee randomly decides which} team takes the kick-off to start the match. 
>
>add "the" in front of "referee"
>----------------------------------------------------------------------
>In  Section_I/law8.tex :
>> @@ -47,9 +47,9 @@
> Before a kick-off at the start of the match or extra time 
> 
> \begin{itemize}
>-\item a coin is tossed and the team that wins the toss decides which goal it will attack in the first half of the match.
>-\item the other team takes the kick-off to start the match. 
>-\item the team that wins the toss takes the kick-off to start the second half of the match.
>+      \item \removed{a coin is tossed and the team that wins the toss decides which goal it will attack} \added{referee randomly decides which team will attack which goal} in the first half of the match.
>+      \item \removed{the other} \added{referee randomly decides which} team takes the kick-off to start the match. 
>+      \item the team that \removed{wins the toss} \added{is other from team taking kick-off at first half} takes the kick-off to start the second half of the match.
>
>Use the wording from the virtual rule book here: "the team that was not given kick-off in the first half of the match takes the kick-off to start the second half of the match."
>—
>Reply to this email directly,  view it on GitHub , or  unsubscribe .
>Triage notifications on the go with GitHub Mobile for  iOS or  Android .
>You are receiving this because you authored the thread. Message ID:  <RoboCup-Humanoid-TC/Rules/pull/20/review/870526001 @ github . com> 
 
 
--
A B
Отправлено из Почты  Mail.ru

**Comment by MaikePaetzel** [comment]
I don't see the commit that resolves these issues?

**Comment by MaikePaetzel** [comment]
According to the rule voting G01Q09, there was no majority in favor of this, so this PR will be rejected.

**Pull request closed without merging**